Formula/s/sugarjar.rb

```diff
@@ -24,10 +24,11 @@ class Sugarjar < Formula
   uses_from_macos "libffi"
 
   def install
+    ENV["BUNDLE_FORCE_RUBY_PLATFORM"] = "1"
     ENV["BUNDLE_VERSION"] = "system" # Avoid installing Bundler into the keg
+    ENV["BUNDLE_WITHOUT"] = "development test"
     ENV["GEM_HOME"] = libexec
 
-    system "bundle", "config", "set", "without", "development", "test"
```

Comment:
This is in `brew create` for Ruby formulae. If we are going to move away from this, we should change that there as well.

Reply:
I think we can just converging to use bundle env vars.